src/main/configurations/Translate/Configuration_PatchRelevanteAndereZaakDelete.xml

```diff
@@ -0,0 +1,108 @@
+            <Exits>
+                <Exit name="EXIT" state="SUCCESS"/>
+                <Exit name="EXCEPTION" state="ERROR"/>
+            </Exits>
+
+            <SenderPipe name="GetAuthorizationHeaderForZakenApi"
+                getInputFromFixedValue="&lt;dummy/&gt;">
+                <IbisLocalSender
+                    name="GetAuthorizationHeaderForZakenApiSender"
+                    javaListener="GenerateAuthorizationHeaderForZakenApi"
+                    returnedSessionKeys="Error,Authorization">
+                </IbisLocalSender>
+                <Forward name="success" path="CheckIfSingleAndereUrl" />
+                <Forward name="exception" path="EXCEPTION" />
+            </SenderPipe>
+
+            <XmlIfPipe name="CheckIfSingleAndereUrl"
+                getInputFromFixedValue="&lt;dummy/&gt;"
+                xpathExpression="$AndereUrlToDelete">
+                <Param name="AndereUrlToDelete" sessionKey="AndereUrlToDelete"/>
+                <Forward name="then" path="CheckIfZaakRelatedToAndereZaak" />
+                <Forward name="else" path="CreatePatchZaakBodyToDeleteAndereZaak" />
+            </XmlIfPipe>
+
+            <XmlIfPipe name="CheckIfZaakRelatedToAndereZaak"
+                getInputFromSessionKey="originalMessage"
+                xpathExpression="/ZgwZaak/relevanteAndereZaken/url = $AndereUrlToDelete">
+                <Param name="AndereUrlToDelete" sessionKey="AndereUrlToDelete"/>
+                <Forward name="then" path="CreatePatchZaakBodyToDeleteAndereZaak" />
+                <Forward name="else" path="LogWarningZaakNotRelated" />
+            </XmlIfPipe>
+
+            <SenderPipe
+                name="LogWarningZaakNotRelated"
+                >
+                <LogSender
+                    name="LogWarningSender" />
+                <Forward name="success" path="EXIT" />
+            </SenderPipe>
+
+            <XsltPipe name="CreatePatchZaakBodyToDeleteAndereZaak"
+                getInputFromSessionKey="originalMessage"
+                styleSheetName="CreeerZaak_LK01/xsl/CreateZgwAndereZaakBodyToDelete.xslt"
+                >
+                <Param name="AndereUrlToDelete" sessionKey="AndereUrlToDelete"/>
+                <Param name="AndereUrlsToDelete" sessionKey="AndereUrlsToDelete" type="DOMDOC"/>
+                <Forward name="success" path="CheckIfRelevanteAndereZakenListEmpty"/>
+            </XsltPipe>
+
+            <XmlIfPipe name="CheckIfRelevanteAndereZakenListEmpty"
+                xpathExpression="/container/relevanteAndereZaken[1]/url">
+                <Forward name="then" path="XmlToJson"/>
+                <Forward name="else" path="CreateEmptyRelevanteAndereZakenJson"/>
+            </XmlIfPipe>
+
+            <XsltPipe name="CreateEmptyRelevanteAndereZakenJson"
+                styleSheetName="UpdateZaak_LK01/xsl/CreateEmptyRelevanteAndereZakenJson.xslt">
+                <Forward name="success" path="PatchZaak" />
+            </XsltPipe>
```

Comment:
Replace with DataSonnetPipe. Can copy/paste the json part from the xslt and stick it in a .jsonnet file. This makes sure it is actually JSON under the hood for the framework and are actually usable by future DataSonnetPipe manipulations. Otherwise it will be a considered a string.